src/elements/List/ListList.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getElementType,
+  getUnhandledProps,
+  META,
+  useKeyOnly,
+} from '../../lib'
+
+function ListList(props) {
+  const { children, className } = props
+
+  const ElementType = getElementType(ListList, props)
+  const classes = cx(
+    className,
+    useKeyOnly(ElementType !== 'ul' && ElementType !== 'ol', 'list'),
```

Comment:
Per the docs the "list" class may be omitted "as a convenience" when using an "ol"  or "ul" tag. Though, I don't think it is required to be omitted. We can likely always have the "list" class present. 

I suspect we can also have the "item" class on all items as well. Let's try always having the classes first. That way, the components are more simple.